lib_eio_windows/test/test_fs.ml

```diff
@@ -119,80 +114,61 @@ let test_append env () =
   Path.save ~create:`Never ~append:true test_file "2nd-write";
   Alcotest.(check string) "append" "1st-write-original2nd-write" (Path.load test_file)
 
-let test_mkdir env () =
-  let cwd = Eio.Stdenv.cwd env in
-  try_mkdir (cwd / "subdir");
-  try_mkdir (cwd / "subdir\\nested");
-  let test_file = cwd / "subdir\\nested\\test-file" in
-  Path.save ~create:(`Exclusive 0o600) test_file "data";
-  Alcotest.(check string) "mkdir" "data" (Path.load test_file);
-  Unix.unlink "subdir\\nested\\test-file";
-  Unix.rmdir "subdir\\nested";
-  Unix.rmdir "subdir"
-
-let test_mkdirs env () =

```

Comment:
As mentioned in the previous PR, could we add this test to the bottom of the file alongside the other tests instead of removing it. Thanks!